app/controllers/items_controller.rb

```diff
@@ -18,19 +18,29 @@ def update
     @merchant = Merchant.find(params[:merchant_id])
     @item = @merchant.items.find(params[:id])
   
-    if @item.update(item_params)
+
+    if params[:disable] == 'true'
+      @item.disabled!
+      flash[:success] = "Item successfully disabled."
```

Comment:
I'm wondering if this line needs to be tested.  SimpleCov is showing that this line is not covered.  I'll check the spec.